tests/lib/openstack_query/runners/conftest.py

```diff
@@ -0,0 +1,18 @@
+from unittest.mock import MagicMock
+import pytest
+
+
+@pytest.fixture(scope="module", name="mock_connection")
+def mock_connection_fixture():
+    """
+    Returns a mocked OpenstackConnection class
+    """
+    return MagicMock()
+
+
+@pytest.fixture(scope="module", name="mock_openstack_connection")
```

Comment:
These should be `function` scope at most, else the same mocks will pass later tests for an `assert_called`